packages/components/src/tools-panel/types.ts

```diff
@@ -49,26 +49,16 @@ export type ToolsPanelProps = {
 	__experimentalLastVisibleItemClass?: string;
 };
 
-export type ToolsPanelHeaderProps = {
+export type ToolsPanelDropdownMenuProps = {
 	/**
 	 * Text to be displayed within the panel header. It is also passed along as
 	 * the `label` for the panel header's `DropdownMenu`.
 	 */
 	label: string;
-	/**
-	 * The `resetAll` prop provides the callback to execute when the "Reset all"
-	 * menu item is selected. Its purpose is to facilitate resetting any control
-	 * values for items contained within this header's panel.
-	 */
-	resetAll: ResetAll;
-	/**
-	 * This is executed when an individual control's menu item is toggled. It
-	 * will update the panel's menu item state and call the panel item's
-	 * `onSelect` or `onDeselect` callbacks as appropriate.
-	 */
-	toggleItem: ( label: string ) => void;
 };
 
+export type ToolsPanelHeaderProps = ToolsPanelDropdownMenuProps;
```

Comment:
Nit: we'd probably need to duplicate the type here, since we'd want the description for the `label` prop to be different for the two components